### DISCUSSION TITLE
Bus doesn't land like expected

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3365] [General]
Since two or three of the latest FBW development mods with experimental FBW the plane doesn't "autoland" anymore: the voice counts down: 100, 50, 40, 30,,20... With 20 I pull back thrust but the plane is starting now or maybe it is a flaring, but it doesn't sink anymore. I have to take off the autopilot and press down the aircraft slightly  for touching the runway and landing.
Is it a correct behaviour or do I have something to prepar with trim? I can't remember this having before. Thanks!

**Comment by robinottawa** [comment]
This happened to me yesterday but I realised I forgot to lower the landing gear. Today it landed okay. 
By the way is it ILS or RNAV?

**Reply by KaiBVG** [reply]
It was ILS.

**Reply by robinottawa** [reply]
ILS, ok. I'm going to try it and watch 320 Sim Pilot's youtube video. There are so many changes being made every week that the tutorials are not necessarily up-to-date. I think it should flare but continue sinking, but I also think I'm supposed to do *something* with the autothrottle, autopilot, etc. ILS should work perfectly. RNAV is still a dog's breakfast for me and I'm still not sure if it's me or the sim or the a/c.

EDIT: 320 Sim Pilot says you can turn off AP on short final but don't have to. But you do have to reduce thrust levers to idle so at about 100' (before you touch down otherwise they will spool up again). I'm trying it now.

EDIT2: It worked as he said. So throttle to idle, AP off, land manually, turn on reverse thrusters. What could go wrong?  :)

EDIT3: (all for future reference) I realize now that if certain CAT autoland capabilities are present, it will do the entire landing for you.

**Reply by KaiBVG** [reply]
Fine, thanks!

**Comment by sidnov** [comment]
Alternatively, you can check out the Custom FBW branch, which provides a much more realistic hand-flying experience. Do note that it is still in an experimental phase, and has bugs.

**Comment by MapleNAV-TCASFAULT** [comment]
If you're using the experimental version from the installer it only has custom FBW and does not feature autoland! If you want the version with autoland (which also includes CFBW) hit this link: https://flybywiresim-packages.nyc3.cdn.digitaloceanspaces.com/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip (Note: the FMA callouts on the PFD are currently incorrect)
